chpl/chpl-api/e2e/script/newman-run-dev.js

```diff
@@ -104,6 +113,7 @@ const jobs = [
     cb => newman.run(statusControllerTests,cb),
     cb => newman.run(testinglabControllerTests,cb),
     cb => newman.run(searchviewControllerTests,cb),
+	cb => newman.run(surveillanceControllerTests,cb),
```

Comment:
Looks like has 2 extra spaces in this line. Same with other newman files.

Reply:
pushed a commit to fix that